### DISCUSSION TITLE
Models should not send an update to the server if the property isn't dirty

### DISCUSSION THREAD

**Discussion by kirkbushell** [livewire/livewire#7836] [3. Feature ideas]
Experiencing some weirdness and unnecessary requests - but perhaps I missed something in the documentation.

Basically, all my forms are updated live - each text input is on blur, each radio/checkbox is on change or live.etc. 

The point is, there's a lot of potential requests here, especially if you just click on and off elements all day. This is fine for radio/checkbox, but if no update is actually made (ie. the property is not "dirty"), then no update should be sent to the server.

I could have sworn that's how it worked in Livewire 2, but maybe I'm wrong.

**Comment by calebporzio** [comment]
Yeah, interesting. I think I agree. Right now livewire just naively triggers a request on the blur event, but ideally it knows if it's dirty and only serves then.

Mind getting a PR started with a failing disk test so we can explore further?

**Reply by joshhanley** [reply]
*dusk
